spec/user-stories.md

```diff
@@ -27,6 +27,12 @@
   *Context:* This improves usability for managing large media or data files.  
   *Issues:* [#18](https://github.com/w3c/lws-ucs/issues/18)
 
+- **<dfn>Discoverable Resource Listings</dfn>**
+
+  **As a** user, **I want** the ability to discover all resources defined in my storage, **so that** I can run (external) indexing  mechanisms.  
```

Comment:
```suggestion
  **As a** user, **I want** the ability to discover all resources defined in my storage, **so that** I can run (external) indexing mechanisms.
```

Reply:
same double space comment at the end of the line here, I'll fix the rogue space between "indexing" and "mechanisms" though